apps/api/views/profiles.py

```diff
@@ -44,14 +45,30 @@ class TestStudentViewSet(ModelViewSet):
     serializer_class = serializers.profiles.TestStudentSerializer
 
 
+def make_ordinal(n):
+    '''
```

Comment:
docstring on same line as quotes, quotes should be double quotes

Reply:
✅